mtp_cashbook/templates/cashbook/includes/batch_buttons.html

```diff
@@ -1,5 +1,7 @@
 {% load i18n %}
 <div>
-  <button type="submit" name="submit" value="submit" class="button" id="tour-step6">{% trans "I've credited ticked payments in NOMIS" %}</button>
+  {% if object_list %}
+    <button type="submit" name="submit" value="submit" class="button" id="tour-step6">{% trans "I've credited ticked payments in NOMIS" %}</button>
+  {% endif %}
```

Comment:
Do we need the `if` here? You're already including this in `transaction_batch_list` inside an `if object_list`…